schema/CONFIG.md

```diff
@@ -0,0 +1,42 @@
+
+### config.yaml schema
+
+Below schema should be used when preparing a `config.yaml` file for models using the tool. Some parameters are optional and marked as such.
+
+<pre><code><b>project_id</b>: [project ID]
```

Comment:
I get that you want to add emphasis to text in a code block and using html tags is the only way, just wondering if putting things in a code block is truly necessary.

If it is, nit: indent the stuff in the pre block so it looks like
```html
<pre>
    [stuff]
</pre>
```

Reply:
Also, not sure what `<code>` does.

`<pre>stuff</pre>`
<pre>stuff</pre>

`<pre><code>stuff<code></pre>`
<pre><code>stuff<code></pre>

Reply:
`<pre><code>...</pre></code>` should [allow](https://stackoverflow.com/a/4611735) for code that doesn't wrap around, but looking at GitHub's markdown rendering, it doesn't make any difference. Updated with `<pre>...</pre>`. 

Also, re: indenting, the <pre> tags honor spaces and tabs as is, so formatting code will result in extra indentation which will be rendered as spaces in markdown. Let me know if that's still an issue and I can look into other formats for the docs. Any suggestions?

Reply:
ohh interesting. got it, nvm then